python-lib/dku_stress_test_center/stress_test_center.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pandas as pd
+from dku_stress_test_center.utils import DkuStressTestCenterConstants, get_stress_test_name
+from drift_dac.perturbation_shared_utils import Shift, PerturbationConstants
+from sklearn.metrics import accuracy_score, f1_score, balanced_accuracy_score
+import logging
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO, format='Stress Test Center Plugin | %(levelname)s - %(message)s')
+
+
+class StressTestConfiguration(object):
+    def __init__(self,
+                 shift_type: Shift,
+                 list_of_features: list = None):
+        self.shift = shift_type
+        self.features = list_of_features  # unused then?
+        # check valid configurations
+
+
+class StressTestGenerator(object):
+    def __init__(self,
+                 config_list: list,  # list of StressTestConfiguration
+                 selected_features: list,
+                 is_categorical: np.array = None,  # wrt selected features
+                 is_text: np.array = None,  # wrt selected features
+                 clean_dataset_size=DkuStressTestCenterConstants.CLEAN_DATASET_NUM_ROWS,
+                 random_state=65537):
+
+        self.config_list = config_list
+        self.selected_features = selected_features
+        self.is_categorical = is_categorical
+        self.is_text = is_text
+        self.is_numeric = None
+
+        self._random_state = random_state
+        self._clean_dataset_size = clean_dataset_size
+
+    def _subsample_clean_df(self,
+                            clean_df: pd.DataFrame):
+
+        np.random.seed(self._random_state)
+
+        return clean_df.sample(n=self._clean_dataset_size, random_state=self._random_state)
+
+    def fit_transform(self,
+                      clean_df: pd.DataFrame,
+                      target_column: str = DkuStressTestCenterConstants.TARGET):
```

Comment:
not sure why we want a default value here ?

Reply:
Agree with Léo